spec/features/vote_spec.rb

```diff
@@ -0,0 +1,64 @@
+require "rails_helper"
+
+feature "user votes on review", %Q{
```

Comment:
Use `%` instead of `%Q`.<br>Use `%Q` only for strings that contain both single quotes and double quotes, or for dynamic strings that contain double quotes.